AzureSpatialAnchorsSample/.gitignore

```diff
@@ -0,0 +1,62 @@
+# This .gitignore file should be placed at the root of your Unity project directory
```

Comment:
It's my understanding that this .gitignore template was built to be put at the root of each Unity project. A lot of the paths are relative, based on that assumption. We could build a new .gitignore and update some the relative paths, but I thought it might be more simple to keep using the default ones, one per project.